iati-activities-schema.xsd

```diff
@@ -2519,17 +2506,19 @@
     </xsd:complexContent>
   </xsd:complexType>
 
-<!-- resultLocationBase extends locationBase to include the common documentation for location as a sub-element of the iati-activity level result element -->
+<!-- resultLocationBase provides the common ref attribute for all location elements in the iati-activity/results element. -->
   <xsd:complexType name="resultLocationBase">
-    <xsd:complexContent>
-      <xsd:extension base="locationBase">
-        <xsd:annotation>
-          <xsd:documentation xml:lang="en">
-            A location already defined and described in the iati-activity/location element.
-          </xsd:documentation>
-        </xsd:annotation>
-      </xsd:extension>
-    </xsd:complexContent>
+    <xsd:sequence>
+      <xsd:any namespace="##other" processContents="lax" minOccurs="0" maxOccurs="unbounded"/>
+    </xsd:sequence>
+    <xsd:attribute name="ref" use="optional" type="xsd:string">
+      <xsd:annotation>
+        <xsd:documentation xml:lang="en">
+          A cross-reference to the internal reference assigned to a defined location: iati-activity/location/@ref.
```

Comment:
The original escaped the `@` character - i.e. `iati-activity/location/\@ref` - perhaps @hayfield can comment on if this escaping is necessary...?  For reference, the previously escaped code did appear correctly when the 2.03 docs site was generated:

![image](https://user-images.githubusercontent.com/8247168/36204267-86747f44-1182-11e8-864c-a16528028d45.png)